### PR TITLE
Update set-up-snowflake-oauth.md

### PR DESCRIPTION
**Pull request by patkearns10**
Ran into an issue with a customer thinking they had whitelisted IP addresses, but had not because it was not applied to the security integration

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md [hunk 1]
```diff
@@ -105,3 +105,10 @@ This error might be because of a configuration issue in the Snowflake OAuth flow
 If you experience a 500 server error when redirected from Snowflake to dbt Cloud, double check that you have whitelisted [dbt Cloud's IP addresses](/docs/cloud/about-cloud/regions-ip-addresses) on a Snowflake account level.
 
 Enterprise customers who have single-tenant deployments will have a different range of IP addresses (network CIDR ranges) to whitelist.
+
+Depending on how you have configured your Snowflake network policies / IP whitelisting - you may have to explicitly add the network policy (that has dbt Cloud IPs whitelisted) to the security integration that you've just created.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md]:
```suggestion
Depending on how you've configured your Snowflake network policies or IP allow listing, you may have to explicitly add the network policy that includes the allow listed dbt Cloud IPs to the security integration you just made.
```

File: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md [hunk 2]
```diff
@@ -105,3 +105,10 @@ This error might be because of a configuration issue in the Snowflake OAuth flow
 If you experience a 500 server error when redirected from Snowflake to dbt Cloud, double check that you have whitelisted [dbt Cloud's IP addresses](/docs/cloud/about-cloud/regions-ip-addresses) on a Snowflake account level.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md]:
```suggestion
If you experience a 500 server error when redirected from Snowflake to dbt Cloud, double-check that you have allow listed [dbt Cloud's IP addresses](/docs/cloud/about-cloud/regions-ip-addresses) on a Snowflake account level.
```

File: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md [hunk 3]
```diff
@@ -105,3 +105,10 @@ This error might be because of a configuration issue in the Snowflake OAuth flow
...
 
 Enterprise customers who have single-tenant deployments will have a different range of IP addresses (network CIDR ranges) to whitelist.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md]:
```suggestion
Enterprise customers who have single-tenant deployments will have a different range of IP addresses (network CIDR ranges) to allow list.
```

**Review by mirnawong1 — COMMENTED** [review]
hey @patkearns10 - have added some small suggestions to this (and old content too!) - let me know what you think!

**Comment by patkearns10** [comment]
> hey @patkearns10 - have added some small suggestions to this (and old content too!) - let me know what you think!

Perfect - thank you!

**Review by mirnawong1 — APPROVED** [review]
lgtm @patkearns10 - thank you

**Pull request merged**